yetanotherco/aligned_layer
# chore: update risc0 version in validating public inputs example

# Description

- Bumps up risc0 version used in validating public inputs example.
- Bumps up sdk version used in validating public inputs example.

# To Test
- Run `generate_risc_zero_fibonacci_proof` in `examples/validating-public-input` and make sure it works properly.
- Run `make submit_fibonacci_proof` in `examples/validating-public-input` and make sure it works properly.
